### ISSUE TITLE
Ensembl gene annotations not uniform

### ISSUE THREAD

**Issue by aiqc** [biothings/mygene.info#138]
If it's possible for multiple ensembl values to be returned, then they should always be a list.

When using `fields="symbol,ensembl.type_of_gene"`
```
[
{
    'query': 'ENSG00000283341',
    '_id': 'ENSG00000283341',
    '_version': 1,
    'ensembl': {'type_of_gene': 'lncRNA'}
},
{
    'query': 'ENSG00000199352', 
    '_id': '100169751',
    '_version': 2, 
    'ensembl': [{'type_of_gene': 'rRNA'}, {'type_of_gene': 'rRNA'}], 
    'symbol': 'RNA5S1'
}
]
```

**Comment by aiqc** [comment]
Also, why no null value for symbol?

**Comment by newgene** [comment]
@aiqc I think this issue can be resolved as described in #42, particularly [this comment](https://github.com/biothings/mygene.info/issues/42#issuecomment-415618382).
